[2024-05-29T00:52:05.581+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-29T00:52:05.623+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dados_climaticos_estruturados.extrair_dados scheduled__2024-05-21T23:30:00+00:00 [queued]>
[2024-05-29T00:52:05.643+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dados_climaticos_estruturados.extrair_dados scheduled__2024-05-21T23:30:00+00:00 [queued]>
[2024-05-29T00:52:05.644+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-29T00:52:05.683+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extrair_dados> on 2024-05-21 23:30:00+00:00
[2024-05-29T00:52:05.695+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=7157) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-29T00:52:05.699+0000] {standard_task_runner.py:63} INFO - Started process 7173 to run task
[2024-05-29T00:52:05.700+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dados_climaticos_estruturados', 'extrair_dados', 'scheduled__2024-05-21T23:30:00+00:00', '--job-id', '2364', '--raw', '--subdir', 'DAGS_FOLDER/weather/custom_dags/dados_climaticos_estruturados.py', '--cfg-path', '/tmp/tmp67xhwyr_']
[2024-05-29T00:52:05.703+0000] {standard_task_runner.py:91} INFO - Job 2364: Subtask extrair_dados
[2024-05-29T00:52:05.786+0000] {task_command.py:426} INFO - Running <TaskInstance: dados_climaticos_estruturados.extrair_dados scheduled__2024-05-21T23:30:00+00:00 [running]> on host 0473b835f7f4
[2024-05-29T00:52:05.929+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dados_climaticos_estruturados' AIRFLOW_CTX_TASK_ID='extrair_dados' AIRFLOW_CTX_EXECUTION_DATE='2024-05-21T23:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-21T23:30:00+00:00'
[2024-05-29T00:52:05.932+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-29T00:52:05.934+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-29T00:52:05.935+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-29T00:52:05.950+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dados_climaticos_estruturados, task_id=extrair_dados, run_id=scheduled__2024-05-21T23:30:00+00:00, execution_date=20240521T233000, start_date=20240529T005205, end_date=20240529T005205
[2024-05-29T00:52:06.004+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-29T00:52:06.028+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-29T00:52:06.031+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
